CHANGELOG.md

```diff
@@ -6,6 +6,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.17.0] - 2022-02-04
+
+### Added
+
+- Get event context on EventSubscription ([#423](https://github.com/paritytech/subxt/pull/423))
+
+### Changed
```

Comment:
isn't the jsonrpsee change missing from here or excluded from the release?

Reply:
Ah good point; the default sort order of PRs is annoying; perhaps it's worth noting in the release doc to filter by `is:pr sort:updated-desc is:closed` to see PRs in order of when they were updated.

Everything else looks good I think; just that one that's missing

Reply:
yupp, makes sense